### ISSUE TITLE
Revise Coefficient Image(s) for Clarity

### ISSUE THREAD

**Issue by pynchmeister** [zapproject/docs#8]
![unknown](https://user-images.githubusercontent.com/33232379/89657708-7b657a80-d89b-11ea-9cd2-0809e8721e65.png)

The image above is confusing, the polynomials need to be uniform. In a nutshell the subscript notation needs improvement. A suggestion to enhance clarity is to put the arrays on separate lines.

Here is the link to the page where the image lives: https://zapproject.gitbook.io/zapproject/oracles/endpoints-bonding-curve/curve-encoding

**Comment by BrianBalayon** [comment]
# Revise Coefficient Notation for Clarity

## The Issue
At first glance the c<sub>1</sub>'s look like supposed to be with n<sub>1</sub> and l<sub>1</sub>, c<sub>2</sub> with n<sub>2</sub> and l<sub>2</sub>, so on and so forth.

## A proposal
The n<sub>i</sub> and l<sub>i</sub> notation are fine.
The c<sub>j</sub> notation could become c<sub>(i,j)</sub>
Where `i` corresponds to the polynomial
And `j` corresponds to the term of the coefficient of the polynomial

## Side Note
I didn't see the current documentation mention the following:
l<sub>i</sub> < l<sub>i+1</sub>
It is more or less implies by an example

**Comment by pynchmeister** [comment]
@Lloyd1553 did that call clarify things for you? Many thanks to @BrianBalayon

**Comment by Lloyd1553** [comment]
yes, it gave me a better understanding of what was the issue was and how to revise it

**Comment by pynchmeister** [comment]
Perfect, I look forward to viewing your creation! 👾
